_partials/_livesync-configure-source-database.md

```diff
@@ -0,0 +1,45 @@
+<Procedure>
+
+1. **Tune the Write Ahead Log (WAL) on the PostgreSQL source database**
+
+   ```sql
+   psql $SOURCE -c "SET wal_level=’logical’;"
+   psql $SOURCE -c "SET max_wal_sender=10;"
```

Comment:
This will be different on AWS RDS/Auora, you can follow  "From AWS RDS/Aurora"/"Tune your source database" from live-migration.

Reply:
Added the AWS section from live migration docs.